include/openssl/core_numbers.h

```diff
@@ -481,6 +487,8 @@ OSSL_CORE_MAKE_FUNC(int, OP_signature_digest_verify_final,
                     (void *ctx, const unsigned char *sig, size_t siglen))
 OSSL_CORE_MAKE_FUNC(void, OP_signature_freectx, (void *ctx))
 OSSL_CORE_MAKE_FUNC(void *, OP_signature_dupctx, (void *ctx))
+OSSL_CORE_MAKE_FUNC(int, OP_signature_get_params, (const OSSL_PARAM params[]))
```

Comment:
Same here, I notice that in the documentation for signatures the `const` is correctly omitted.

Reply:
Fixed